docs/variants/msi_z690/heads.md

```diff
@@ -0,0 +1,190 @@
+- [Intel Management Engine based on v16.1.30.2307 revision d0b63476](https://github.com/Dasharo/dasharo-blobs/blob/d0b63476/msi/ms7d25/me.bin)
+- [Intel Flash Descriptor based on v1.2 revision d0b63476](https://github.com/Dasharo/dasharo-blobs/blob/d0b63476/msi/ms7d25/descriptor.bin)
+- [Intel Firmware Support Package based on RPL-S C.0.BD.40 revision 481ea7cf](https://github.com/intel/FSP/tree/481ea7cf/RaptorLakeFspBinPkg/Client/RaptorLakeS)
+- [Intel microcode based on ADL/RPL C0/H0 0x0000002e revision microcode-20230808](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/tree/microcode-20230808/intel-ucode/06-97-05)
+- [Intel microcode based on RPL B0 0x00000119 revision microcode-20230808](https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files/tree/microcode-20230808/intel-ucode/06-b7-01)
+
+[newsletter]: https://newsletter.3mdeb.com/subscription/D7dQvGx6k
+[msi_ms7d25_v0.9.0_ddr4_heads.rom_hash]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7d25/heads/v0.9.0/msi_ms7d25_v0.9.0_ddr4_heads.rom.sha256
+[msi_ms7d25_v0.9.0_ddr4_heads.rom_sig]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7d25/heads/v0.9.0/msi_ms7d25_v0.9.0_ddr4_heads.rom.sha256.sig
+[msi_ms7d25_v0.9.0_ddr5_heads.rom_hash]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7d25/heads/v0.9.0/msi_ms7d25_v0.9.0_ddr5_heads.rom.sha256
+[msi_ms7d25_v0.9.0_ddr5_heads.rom_sig]: https://dl.3mdeb.com/open-source-firmware/Dasharo/msi_ms7d25/heads/v0.9.0/msi_ms7d25_v0.9.0_ddr5_heads.rom.sha256.sig
+
+## Hardware Configuration Matrix
+
+Generally, the same [hardware configuration](hardware-matrix.md) as for the
+UEFI variant applies.
+
+A notable addition is usage of the
+[Nitrokey 3A Mini](https://shop.nitrokey.com/shop/nk3am-nitrokey-3a-mini-149)
+USB device, which is required for Heads installation and usage.
+
+## Test Matrix
+
+Please refer to the [tests results spreadsheet](https://docs.google.com/spreadsheets/d/1yWZ--zFPIsQhXZByf7nJIrasQYuRSf1yCi60lY_RGsQ/edit#gid=5649308).
+
+## Building manual
+
+This section presents the crucial steps required to build the Dasharo Heads
+firmware. For more information, you may also refer to the official
+[Heads building documentation](https://osresearch.net/general-building/).
+
+### Requirements
+
+This guide was verified on Ubuntu 22.04. In practice, any Linux distribution
+with [Docker](https://www.docker.com/) support should be enough to complete it.
+
+Make sure that you have following packages installed:
+
+- Docker
+    + follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    + follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+- Git
+
+    ```bash
+    sudo apt -y install git
+    ```
+
+### Building
+
+1. Clone Dasharo Heads repository
+
+   ```bash
+    git clone https://github.com/Dasharo/heads.git
+   ```
+
+2. Navigate to the source code directory and checkout to the desired revision:
```

Comment:
It renders as '1.' in my local preview for both platforms.

Reply:
Fixed [152a450](https://github.com/Dasharo/docs/pull/762/commits/152a450d41fde23922d75d58e3d57fbc6ab9c1f2)